### PR TITLE
CocoaPods 1.8 Beta Blog Post

### PR DESCRIPTION
**Pull request by dnkoutso**

### CODE REVIEW

File: _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+```
+pod repo remove master
+```
+
+For more information regarding CDN, please read our previous blog post [here](http://blog.cocoapods.org/CocoaPods-1.7.2/)!
+
+### `info_plist` Podspec DSL
+
+TODO
```
Comment (dnkoutso) [inline _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown]: @amorde can you provide a paragraph and any screenshots for this? I can write something up instead.

File: _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+categories: cocoapods releases
+---
+
+_CocoaPods 1.8_ switches to CDN as the default and comes with a few enhancements!
+
+<!-- more -->
+
+This is an exciting release as the [master specs repo](https://github.com/CocoaPods/Specs) is no longer required to be cloned to start using CocoaPods!
+
+### CDN as Default
```
Comment (dnkoutso) [inline _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown]: @igor-makarov does this all make sense?

Comment (igor-makarov) [reply _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown]: I think we might wanna clarify that 1.7.0 can't actually work with the current CDN, just in case.

Comment (dnkoutso) [reply _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown]: sounds good.

File: _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown [hunk 3]
```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title:  "CocoaPods 1.8 Beta is Here!"
+author: dnkoutso
+categories: cocoapods releases
+---
+
+_CocoaPods 1.8_ switches to CDN as the default and comes with a few enhancements!
```
Comment (segiddins) [inline _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown]: `the CDN as the default spec repo source`

File: _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+_CocoaPods 1.8_ switches the CDN as the default spec repo source and comes with a few enhancements!
+
+<!-- more -->
+
+This is an exciting release as the [master specs repo](https://github.com/CocoaPods/Specs) is no longer required to be cloned to start using CocoaPods!
+
+### CDN as Default
+
+CDN support was first introduced in the [1.7 release](http://blog.cocoapods.org/CocoaPods-1.7.0-beta/) and aims to speed up initial setup and dependency analysis dramatically. With 1.8, CocoaPods no longer requires to clone the [master specs repo](https://github.com/CocoaPods/Specs) in order to function which has become huge since the inception of the project.
```
Comment (paulb777) [inline _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown]: cloning the now huge [master specs repo](https://github.com/CocoaPods/Specs).

File: _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+{% breaking_image /assets/blog_img/CocoaPods-1.8.0/project_name.png %}
+</center>
+
+It is up to the consumers to choose their own grouping and provide helper methods in their `Podfile` that automatically apply the project name to use. For example, another grouping idea is to group pods by their platform such as `iOS` or `macOS`.
+
+**Note**: The `project_name` option currently requires the `generate_multiple_pod_projects` installation option to _also_ be enabled in order for it to function. Incremental installation has also been updated to take into consideration the project name used for each pod and will continue to work as expected.
+
+### Test Spec Enhancements
+
+Test specs have become an integral part of CocoaPods and with this release a couple new features have been added.
```
Comment (paulb777) [inline _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown]: s/with this release//

File: _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+
+### What's Next?
+
+CocoaPods 1.8 is a _very_ exciting release, and we're very happy for you to try it out, and recommend you upgrade:
+
+```sh
+$ gem install cocoapods --pre
+```
+
+For future versions, we continue to explore new ways to allow pod authors and pod consumers to configure how to integrate pods into their project, for example by specifying individual package or linkage settings. We have published a [proposal](https://docs.google.com/document/d/1lr2mHkwhNha5_0Wfa_pYL51MAxoro0LBxx_DUzuSu0Y/edit) that we are currently exploring and we welcome your comments!
```
Comment (paulb777) [inline _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown]: to configure integrating pods

File: _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown (hunk 4 above)
Comment (dnkoutso) [reply _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown]: sorry where should I replace this exactly?

Comment (dnkoutso) [reply _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown]: oh I see

File: _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+
+Afterwards, run the following command to remove the [master specs repo](https://github.com/CocoaPods/Specs):
+
+```
+pod repo remove master
+```
+
+If you wish to continue using the `git` based source there is no action to be taken and CocoaPods should continue working as usual.
+
+Upon installing CocoaPods for the first time, the `git` based [master specs repo](https://github.com/CocoaPods/Specs) will _no longer_ be cloned and users may start using CocoaPods almost instantly. Below you can find a video demonstration of integrating a project with CocoaPods 1.8:
```
Comment (amorde) [inline _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown]: Maybe add a note that the CDN will also be used if no source is specified in the Podfile

File: _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown (hunk 2 above)
Comment (dnkoutso) [reply _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown]: done.

File: _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown (hunk 7 above)
Comment (dnkoutso) [reply _posts/2019-08-01-CocoaPods-1.8.0-beta.markdown]: done

**Comment by dnkoutso** [comment]
3 approvals, CocoaPods 1.8.0.beta.1 shipped. Merging.

**Pull request merged**